widgets/htdocs/alphafold/colorHelpers.js

```diff
@@ -0,0 +1,35 @@
+const commonColorsToRGBMap = {
```

Comment:
why do we need all these colour mappings? i get the RGB one but why the Hex one as well. There is a mixture of how the colours are used (its either HEX or RGB)